components/auth/EventOrganizerRegisterForm.tsx

```diff
@@ -29,15 +29,15 @@ function EventOrganizerRegisterForm() {
         username: data.username,
         password: data.password,
         organizationName: data.organizationName,
-        verificationStatus: 'NotVerified'
+        verificationStatus: "NotVerified"
```

Comment:
To be fair, I prefer status to be like "NOT_VERIFIED" (similar to Enums) rather than "NotVerified" but it's not the end of the world :D